com.unity.sg2/Documentation~/Hexagon-Grid-Node.md

```diff
@@ -0,0 +1,23 @@
+# Hexagon Grid Node
+
+## Description
+Generates a hexagonal grid based on input **UV**. The scale of the grid is controlled by input **Scale**. In addition to grid tile outlines, the nodes also outputs the distance from edges, and a unique value or ID for each tile.
+
+Since aliasing can be a concern, the node provides three levels of samples. One sample gives the best performance, but aliasing artifacts may be visible. Using Four or Nine samples will reduce aliasing, but will be slower to generate.
+
+## Ports
+
+| Name        | Direction           | Type  | Binding | Description |
+|:------------ |:-------------|:-----|:---|:---|
+| UV      | Input | Vector 2 | UV | Input UV value |
+| Scale      | Input | Float    | None | Grid scale |
```

Comment:
this should be a vector 2?

Reply:
Yes it should!  Thanks for catching that. I fixed it.